src/main/resources/config/liquibase/changelog/20240829165533_added_additionalInfo_to_Company.xml

```diff
@@ -0,0 +1,18 @@
+<databaseChangeLog
+      xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+      xmlns:pro="http://www.liquibase.org/xml/ns/pro"
```

Comment:
IntelliJ indicates these two are not used.

Reply:
I copy and pasted from liquibase's doc. I guess they need to update them. 😆